### ISSUE TITLE
Stack overflow for openapi 3.0 file

### ISSUE THREAD

**Issue by georgkeller** [landgraf-dev/openapi-delphi-generator#9]
When I try to use the generator on the attached api file, I get a stack overflow:
```
.\OpenApiDelphiGen -i:".\testapi.json" -o:"D:\Delphi\test" -n:testapi -m:MultipleClientsFromFirstTagAndOperationId

OpenApi Client Generator for Delphi version 1.1.1
Copyright (c) Landgraf.dev - all rights reserved.

EStackOverflow: Stack overflow
```
The api file works fine in swagger and other openapi tools.

[testapi.zip](https://github.com/landgraf-dev/openapi-delphi-generator/files/12465499/testapi.zip)

**Comment by wlandgraf** [comment]
Same fix from issue #8 also fixes this.

**Comment by georgkeller** [comment]
> Same fix from issue #8 also fixes this.

I still get an exception for the example file. I used the zip for the release 1.1.4, but the version number is still 1.1.1:

OpenApi Client Generator for Delphi version 1.1.1
Copyright (c) Landgraf.dev - all rights reserved.

EAccessViolation: Access violation at address 00969E85 in module 'OpenApiDelphiGen.exe'. Read of address 00000008
